fighthealthinsurance/ml/ml_models.py

```
@@ -158,6 +182,26 @@ async def generate_chat_response(
+            input_text: The text to extract the entity from
+            entity_type: The type of entity to extract (e.g., 'fax_number', 'insurance_company')
+
+        Returns:
+            Extracted entity or None
+        """
+        result = await self._infer(
+            system_prompts=["You are a helpful assistant."],
+            prompt=f"Extract the {entity_type} from the following text: {input_text}",
+        )
+        if result:
+            if result[1]:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)